app/package.json

```diff
@@ -79,7 +79,7 @@
     "workbox-webpack-plugin": "^4.3.1"
   },
   "scripts": {
-    "start": "BROWSER=none node scripts/start.js",
+    "start": "node scripts/start.js",
```

Comment:
Was this change intentional?

Reply:
Nope!  Removing it now. (Windows problems)